html/elements/col.json

```diff
@@ -40,15 +40,14 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": null
+                "version_added": "1"
               },
               "chrome_android": "mirror",
               "edge": {
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/915'>bug 915</a>."
```

Comment:
the bug was wontfixed? So, I guess it still doesn't work?

Reply:
It's possible, but it'd be interesting for the property to be defined on the interface if it doesn't...

Reply:
I went ahead and reverted these changes so they can be researched at a later time!